src/app/components/Header/index.tsx

```diff
@@ -1,21 +1,29 @@
 'use client'
+import { MENU, SITE_NAME } from '@/app/constants'
 import { Container, Nav, Navbar, Offcanvas } from 'react-bootstrap'
-import { SITE_NAME } from '@/app/constants'
 
 export function Header() {
   return (
     <header>
       <Navbar expand="lg" className="bg-body-tertiary">
         <Container>
-          <Navbar.Brand href="/">{SITE_NAME}</Navbar.Brand>
+          <Navbar.Brand href="/">
+            <h1>{SITE_NAME}</h1>
```

Comment:
`h1` is incorrect here. Brand is not heading. You can change font size of navbar brand using SCSS variables.
https://getbootstrap.com/docs/5.3/components/navbar/#sass-variables

If you are not confident enough with a framework, you should read their documentation more.

Reply:
Ok, I now fixed it with inline bootstrap font size. Thanks.

Reply:
This is not what I requested @YosefAshenafi. Please read what I wrote carefully.